packages/EasyDoctrine/src/Dispatchers/DeferredEntityEventDispatcher.php

```diff
@@ -128,14 +133,21 @@ public function deferUpdate(int $transactionNestingLevel, object $object, array
         $this->entityChangeSets[$transactionNestingLevel][$oid] = $entityChangeSet;
     }
 
-    public function disable(): void
+    public function disable(?array $entityClasses = null): void
     {
-        $this->enabled = false;
+        if ($entityClasses === null) {
+            $this->enabled = false;
+            $this->disabledEntityClasses = [];
+        }
+
+        if ($entityClasses !== null) {
+            $this->disableFor($entityClasses);
```

Comment:
I think we need set the `$this->enabled` value to `true` here.
Let's imagine these two calls:
```
->disable();
->disable([SomeEntity::class]);
````

Reply:
I think we shouldn't. The example above just tries to disable a certain entity while all the entities are disabled. So it shouldn't make any effect I believe.

Reply:
I think that 
```
->disable();
->disable([SomeEntity::class]); 

```
means that dispatching should be disabled for all entitities